book/forms.rst

```diff
@@ -463,173 +463,15 @@ Validation is a very powerful feature of Symfony and has its own
    single: Forms; Validation groups
 
 .. _book-forms-validation-groups:
-
-Validation Groups
-~~~~~~~~~~~~~~~~~
-
-If your object takes advantage of :ref:`validation groups <book-validation-validation-groups>`,
-you'll need to specify which validation group(s) your form should use::
-
-    $form = $this->createFormBuilder($users, array(
-        'validation_groups' => array('registration'),
-    ))->add(...);
-
-If you're creating :ref:`form classes <book-form-creating-form-classes>` (a
-good practice), then you'll need to add the following to the ``setDefaultOptions()``
-method::
-
-    use Symfony\Component\OptionsResolver\OptionsResolverInterface;
-
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
-    {
-        $resolver->setDefaults(array(
-            'validation_groups' => array('registration'),
-        ));
-    }
-
-In both of these cases, *only* the ``registration`` validation group will
-be used to validate the underlying object.
-
-.. index::
-   single: Forms; Disabling validation
-
-Disabling Validation
-~~~~~~~~~~~~~~~~~~~~
-
-.. versionadded:: 2.3
-    The ability to set ``validation_groups`` to false was introduced in Symfony 2.3.
-
-Sometimes it is useful to suppress the validation of a form altogether. For
-these cases you can set the ``validation_groups`` option to ``false``::
-
-    use Symfony\Component\OptionsResolver\OptionsResolverInterface;
-
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
-    {
-        $resolver->setDefaults(array(
-            'validation_groups' => false,
-        ));
-    }
-
-Note that when you do that, the form will still run basic integrity checks,
-for example whether an uploaded file was too large or whether non-existing
-fields were submitted. If you want to suppress validation, you can use the
-:ref:`POST_SUBMIT event <cookbook-dynamic-form-modification-suppressing-form-validation>`.
-
-.. index::
-   single: Forms; Validation groups based on submitted data
-
 .. _book-form-validation-groups:
```

Comment:
We should add labels for all other removed headlines to make the reader aware of where to find the moved information.